### DISCUSSION TITLE
Single file, curated CHANGELOG

### DISCUSSION THREAD

**Discussion by tony** [mobxjs/mobx-state-tree#2030] [Ideas] [answered]
Hi, thank you for the project!

**_Feature request_**
Changelog file

**Is your feature request related to a problem? Please describe.**
<4.0's changelog was better IMO.

**Describe the solution you'd like**
- Single file change log
- Curation from maintainer (order by importance)
- More info: [keepachangelog.com](https://keepachangelog.com/en/1.1.0/)

**Describe alternatives you've considered**
There's no alternative but it seems [changelog.md was removed in favor of releases](https://github.com/mobxjs/mobx-state-tree/blob/02601c3/changelog.md#400) in >4.0 in favor of GitHub Releases.

**Additional context**

<details>

See https://github.com/webpack/webpack/issues/15318 for the rationale / example I've given

GitHub Releases (fragmented, undifferentiated, and in a "blog" subsite detached from any changelog file):

https://user-images.githubusercontent.com/26336/185774527-2150ac43-9a6c-4281-84d4-7ccb5db1924e.mp4

Babel's single file changelog (everything's on one page, at least):

https://user-images.githubusercontent.com/26336/185774539-f02b0c23-493d-4850-b70e-ff162a4f4ff9.mp4

</details>

**Are you willing to (attempt) a PR?**
* [x] Yes

  But a maintainer would be a better for this. Also it would be appropriate to curate releases,

**Comment by tony** [comment]
I should also rewind: I am in a chipper disposition and am happy with the project. Just have a universal distaste for GH Releases.

I've nothing wrong with GH Releases to supplement things, but I'd never suggest it as a replacement for an actual changelog

Example from a project of mine: [CHANGES file](https://github.com/tmux-python/tmuxp/blob/v1.13.0/CHANGES#tmuxp-1130-2022-08-14), [CHANGES on docs site](https://tmuxp.git-pull.com/history.html#tmuxp-1-13-0-2022-08-14), [Release](https://github.com/tmux-python/tmuxp/releases/tag/v1.13.0)

**Comment by coolsoftwaretyler** [comment]
Hey @tony! Sorry no one got back to you for so long. 

I'm going to convert this to a discussion topic, rather than an issue. I don't want to close it out entirely, but I am just sort of getting up to speed on the core team here and I don't have a yes/no for you. 

Thanks for the idea, and apologies once more for the delay!

**Reply by tony** [reply]
@coolsoftwaretyler Thank you!

**Reply by tony** [reply]
Also - I am still interested in a single file changelog.

I will add a link in the main post to https://keepachangelog.com/en/1.1.0/

**Reply by coolsoftwaretyler** [reply, marked as answer]
Hey @tony,

I'm gonna close the loop on this. After running a few releases, I've been very happy with the GitHub process. Apologies that it isn't your preferred outcome, but I think we're going to move forward as is for some time. 

Thanks for putting it out there, though! Really appreciate your time.

**Reply by tony** [reply]
It is fine. Thank you for taking the time to maintain the package, and for getting back to this!
